File: C#_Projects/Okul_Bolumleri/Mekatronik.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Okul_Bolumleri
{
    public class Mekatronik : Anasayfa
    {
        double bolumUcreti = 30000;
        int bursOraniLimiti = 50;
        public override void bolumOzellikleriniGetir()
        {
            Console.WriteLine($"Bu bölümün ücreti { bolumUcreti} TL, maksimum verebileceği burs oranı ise % { bursOraniLimiti}");
        }

        public override void bursluFiyatiHesapla()
        {
            bursluFiyat = bolumUcreti - (bolumUcreti * (bursOrani / 100));

            if (bursOrani > 0 && bursOrani <= bursOraniLimiti)
            {
                Console.WriteLine("Bu bölümün %" + bursOrani + " burslu hali = " + bursluFiyat + " TL'dir.");
            }
            else
            {
                Console.WriteLine("Burs oranı 0'dan aşağıda veya % " + bursOraniLimiti + "'den daha yukarıda olamaz");
            }
        }
    }
}

File: Javascript_Projects/Universite_Bolumleri/mekatronik.js
import { anasayfa } from "./anasayfa.js";

export class mekatronik extends anasayfa {
    constructor(bursOrani) {
        super(bursOrani);
    }

    bolumUcreti = 30000;
    bursOraniLimiti = 50;

    bolum_ozelliklerini_getir() {
        console.log("Mekatronik bölümünün üniversite ücreti = " + this.bolumUcreti + " TL" +
            ", maksimum verebileceği burs oranı ise %" + this.bursOraniLimiti + " olarak gözükmektedir");
    }

    burslu_fiyati_hesapla() {
        this.bursluFiyat = this.bolumUcreti - (this.bolumUcreti * (this.bursOrani / 100));

        if (this.bursOrani > 0 && this.bursOrani <= this.bursOraniLimiti) {
            console.log("Bu bölümün %" + this.bursOrani + " burslu hali = " + this.bursluFiyat + " TL'dir");
        }
        else {
            console.log("Burs oranı %0'dan aşağıda veya %" + this.bursOraniLimiti + "'den daha yukarıda olamaz");
        }
    }

}